src/main/java/life/qbic/io/commandline/AuthenticationOptions.java

```diff
@@ -76,4 +114,42 @@ public String toString() {
         .toString();
     //ATTENTION: do not expose the password here!
   }
+
+  public static class ReadProperties {
+
+    public static TreeMap<String, String> getProperties(String infile) {
+      final int lhs = 0;
+      final int rhs = 1;
+
+      TreeMap<String, String> map = new TreeMap<String, String>();
+      BufferedReader  bfr = null;
+      try {
+        bfr = new BufferedReader(new FileReader(new File(infile)));
+      } catch (FileNotFoundException e) {
+        throw new RuntimeException(e);
+      }
+
+      String line;
+      while (true) {
```

Comment:
NIT: I'd change the while(true) loop to a variable based solution.  I'd introduce a boolean variable hasNextLine and update it in the while loop instead of breaking out of it
```suggestion
      while (hasNextLine) 
      {
      hasNextLine = checkforNextLine()
      Your awesome magic
      }
```